metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/utils/EnrichmentFields.java

```diff
@@ -1,4 +1,4 @@
-package eu.europeana.metis.utils;
+package eu.europeana.enrichment.utils;
 
 import eu.europeana.corelib.definitions.jibx.Contributor;
 import eu.europeana.corelib.definitions.jibx.Coverage;
```

Comment:
This class has easy to solve SonarQube issues.

Reply:
See previous comment about this class.